Reinstate baseline mode; preserve alternative model fitting diagnostics (#164)
epiforecasts/eval-by-method: report/quarto/_methods.qmd

The platform solicited real-time forecasts for between one and four weeks ahead for 32 European countries.
Any forecaster was eligible to participate in the Hub, and there were no selection criteria.

We excluded forecasts that did not report the full set of 23 quantiles, to ensure fair comparison among probabilistic results.

> We excluded the ensemble model created by the Hub team, because it is a function of the contributed forecasts and would therefore double-count them.
> We retained the Hub baseline model, which is an independently specified statistical model and so contributes to the sample on the same terms as any other participant.

Reviewer: ```suggestion
We excluded the ensemble model created by the Hub team, because it is a function of the contributed forecasts and would therefore double-count them, but retained the Hub baseline model as an independently specified statistical model contributing on the same terms as any other participant.
```
